src/test/co_dumps/co_dump_inital.csv

```diff
@@ -29,7 +29,7 @@
 [EffectRack1_EffectUnit4_Effect4],parameter4_set_default,0
 [EffectRack1_EffectUnit2_Effect2],parameter2_link_type,0
 [QuickEffectRack1_[Channel3]_Effect1],parameter5_set_zero,0
-[Master],latency,5.80499
+[App],output_latency_ms,5.80499
```

Comment:
My dump button is broken. Does yours work? I will file a bug.

Reply:
> My dump button is broken. Does yours work? I will file a bug. 

Last time it check it didit creates a file in `~/.mixxx`. It does not create a save file dialog.

Reply:
Oh, right it is working. I just needed to look in my testing .mixxx folder 🤦